### PR TITLE
Updated docstring of exp_smoothing.py

### PR DESCRIPTION
**Pull request by mathco-wf**
##### For all contributions
- [x] I've added myself to the [list of contributors](https://github.com/alan-turing-institute/sktime/blob/main/.all-contributorsrc).
- [ ] Optionally, I've updated sktime's [CODEOWNERS](https://github.com/alan-turing-institute/sktime/blob/main/CODEOWNERS) to receive notifications about future changes to these files.
- [ ] I've added unit tests and made sure they pass locally.

##### For new estimators
- [ ] I've added the estimator to the online documentation.
- [ ] I've updated the existing example notebooks or provided a new one to showcase how my estimator works.

### CODE REVIEW

File: sktime/forecasting/exp_smoothing.py [hunk 1]
```diff
@@ -19,28 +19,39 @@ class ExponentialSmoothing(_StatsModelsAdapter):
...
     Parameters
     ----------
-    trend : str{"add", "mul", "additive", "multiplicative", None}, optional
-    (default=None)
-        Type of trend component.
-    damped_trend : bool, optional (default=None)
+    trend : str or None, default=None
+        Type of trend component.Takes one of 
+        {"add", "mul", "additive", "multiplicative", None}     
+    damped_trend : bool, default=False
```
Comment (SveaMeyer13) [inline sktime/forecasting/exp_smoothing.py:25]: @RHKuhns suggested having options listed directly in the type, i.e.
```suggestion
    trend : {"add", "mul", "additive", "multiplicative", None}, default=None
        Type of trend component.
    damped_trend : bool, default=False
```

**Review by SveaMeyer13 — COMMENTED** [review]
Hi @marco-wf, this looks very good! One minor thing: @RHKuhns suggested having a predefined set of options listed directly in the type (See my comment). We should agree on something here. 

We started changing the author name to the github handle recently. So if you don't mind it would be great if you could change Markus Löning to mloning and remove the "@" from big-o. 

Thank you!

File: sktime/forecasting/exp_smoothing.py [hunk 2]
```diff
@@ -19,28 +19,39 @@ class ExponentialSmoothing(_StatsModelsAdapter):
...
+        {'estimated','heuristic','legacy-heuristic','known',None}.
+        If 'known' initialization is used, then `initial_level` must be
+        passed, as well as `initial_trend` and `initial_seasonal` if
+        applicable.
+        'heuristic' uses a heuristic based on the data to estimate initial
+        level, trend, and seasonal state. 'estimated' uses the same heuristic
+        as initial guesses, but then estimates the initial states as part of
+        the fitting process.
 
     References
```
Comment (SveaMeyer13) [inline sktime/forecasting/exp_smoothing.py:56]: We could add a note somewhere that this is a wrapper around the statsmodel implementation of exponential smoothing, What do you think @mloning, @RNKuhns?

Comment (mathco-wf) [reply sktime/forecasting/exp_smoothing.py:52]: @SveaMeyer13 I did the changes suggested by you, please check.

**Comment by fkiraly** [comment]
Sure.

The general recommendation is to learn how to use GitHub checks and/or local checks.
The issues you have can be easily found and resolved using local linter checls.
If you need help setting these up, feel free to join the weekly community collab session on discord, every Friday 12noon UTC.

You can also look at the checks in GitHub: click "details" on the failing check, and then look at the logs.
In your case, there are two trailing whitespaces that you need to remove. Here's a screenshot of one:
image
it tells you which file and which line.

I've cut off the second one to leave you the thrill of the challenge of finding out how to access the logs 😄

**Comment by mathco-wf** [comment]
@fkiraly The issue was related to trailing whitespaces only, I have rectified the issue. Can you please merge this PR now.

**Review by fkiraly — APPROVED** [review]
alright, looks good now!

**Pull request merged**